models/buffer_disk.go

```diff
@@ -33,17 +33,18 @@ type DiskBuffer struct {
 	isEmpty bool
 }
 
-func NewDiskBuffer(name, alias, path string, stats BufferStats) (*DiskBuffer, error) {
-	pluginName := name
-	if alias != "" {
-		pluginName += "-" + alias
-	}
-
-	filePath := filepath.Join(path, pluginName)
+func NewDiskBuffer(id, path string, stats BufferStats) (*DiskBuffer, error) {
```

Comment:
I would change the order of `name` and `id` but I don't have strong feelings about it. Let me know if you prefer the current way...